certbot/certbot/_internal/main.py

```diff
@@ -113,12 +113,20 @@ def _get_and_save_cert(le_client, config, domains=None, certname=None, lineage=N
         if lineage is not None:
             # Renewal, where we already know the specific lineage we're
             # interested in
-            logger.info("Renewing an existing certificate")
+            logger.info(
+                "Renewing%s an existing certificate for %s",
```

Comment:
What do you think about "Renewing an existing certificate for %s" / "Simulating renewal of an existing certificate for %s"?